### PR TITLE
New package: throttled-0.7

### PR DESCRIPTION
**Pull request by dawidpotocki**
Throttled is a program to fix throttling issues (https://old.reddit.com/r/thinkpad/comments/870u0a/t480s_linux_throttling_bug/) on some Intel laptops like

* Lenovo ThinkPad T480, T480s, X1C5, X1C6, T580, L480, T470, X280, Anniversary Edition 25, E590 w/ RX 550X, P43s, E480, E580
* Dell XPS 9365, 9370, Latitude 7390 2-in-1
* Microsoft Surface Book 2

### CODE REVIEW

File: srcpkgs/throttled/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'throttled'
+pkgname=throttled
+version=0.7
+revision=1
+wrksrc=throttled-${version}
```
Comment (abenson) [inline srcpkgs/throttled/template:5]: This should already be the default

File: srcpkgs/throttled/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'throttled'
+pkgname=throttled
+version=0.7
```
Comment (ericonr) [inline srcpkgs/throttled/template:3]: 0.9.2 is out

File: srcpkgs/throttled/patches/setup.py.patch [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
++
++setup(
++    name='lenovo_fix',
++    version='@VERSION@',
++    description='Workaround for Intel throttling issues in Linux',
++    author='Francesco Palmarini',
++    author_email='palmarini@unive.it',
++    url='https://github.com/erpalma/throttled',
++    license='MIT',
++)
```
Comment (ericonr) [inline srcpkgs/throttled/patches/setup.py.patch:14]: Please upstream this :)

File: srcpkgs/throttled/template [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+pre_build() {
+	sed -i "s/@VERSION@/${version}/" setup.py
+}
+
+post_install() {
+	vinstall mmio.py 644 ${py3_sitelib}/lenovo_fix mmio.py
+	vinstall lenovo_fix.py 644 ${py3_sitelib}/lenovo_fix lenovo_fix.py
+	vconf etc/lenovo_fix.conf
+	vsv lenovo_fix
+	sed -i "s|@PATH@|${py3_sitelib}|" ${DESTDIR}/etc/sv/lenovo_fix/run
```
Comment (ericonr) [inline srcpkgs/throttled/template:25]: Use `vsed`.

File: srcpkgs/throttled/template [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+depends="python3-dbus python3-gobject"
+short_desc="Workaround for Intel throttling issues in Linux"
+maintainer="Dawid Potocki <dawid@dawidpotocki.com>"
+license="MIT"
+homepage="https://github.com/erpalma/throttled"
+distfiles="https://github.com/erpalma/throttled/archive/v${version}.tar.gz"
+checksum=64f139b3fd6f13381105f30ed4598faa69e59bfd613d26dfde29f46578b0b2a1
+
+pre_build() {
+	sed -i "s/@VERSION@/${version}/" setup.py
```
Comment (ericonr) [inline srcpkgs/throttled/template:17]: And `vsed` as well.

**Comment by dawidpotocki** [comment]
I do not use Void Linux anymore and I do not plan to update this PR to
get this package merged.

**Pull request closed without merging**